### ISSUE TITLE
Does it work with financial time series?

### ISSUE THREAD

**Issue by mackancurtaincheeks** [Corleno/HHMM#1]
I can't find any readme or information on this module - does it work and if yes - does it work with financial time series data?

**Comment by Corleno** [comment]
Yes, it should work for finance data. But you need to modify the observation model in HHMM. Current codes mostly focus on  screening test data in healthcare research.
